extensions/map-layers-auth/README.md

```diff
@@ -32,13 +33,14 @@ export function ArcGisOauthRedirect() {
 }
 ```
 
-2. __Client IDs__\
-The hosting application must be registered in either the ArcGIS Online server (cloud offering) or an ArcGIS enterprise server (on-premise). The registered application must then provide it's associated clientID and redirectUri to the `ArcGgisAccessClient` object.  The ui-test-app application provides a complete sample configuration. 
-The maplayers widget has also been updated to support OAuth2: if needed, a popup window will be displayed to trigger the external OAuth process with the remote ArcGIS server. When the process completes, the focus returns to the map-layers widget and layer is ready to be added/displayed.\
+1. __Client IDs__\
+The hosting application must be registered in either the ArcGIS Online server (cloud offering) or in ArcGIS enterprise server (on-premise). The registered application must then provide its associated clientID and redirectUri to the `ArcGisAccessClient` object.  The ui-test-app application provides a complete sample configuration.
```

Comment:
we need a better test app to use for this example imo,

need to see about integrating this pkg in the sandbox, and linking to an example there?

Reply:
Already started a discussion with the sandbox "team", more complicated than I expected to configure.

Reply:
could you loop me in? kinda defeats the purpose of the sandbox if they cant pull in a new pkg into their setup easily